.github/workflows/ember.yml

```diff
@@ -22,41 +23,46 @@ jobs:
     name: "Build"
     runs-on: ubuntu-latest
     outputs:
-      build_num: ${{ steps.tag.outputs.build_num }}
-      image_tag_build: ${{ steps.tag.outputs.image_tag_build }}
-      image_tag_branch: ${{ steps.tag.outputs.image_tag_branch }}
+      build_num: ${{ steps.export_build_tag.outputs.build_num }}
+      image_tag_build: ${{ steps.export_build_tag.outputs.image_tag_build }}
+      image_tag_branch: ${{ steps.export_build_tag.outputs.image_tag_branch }}
     steps:
 
       - name: "Checkout"
+        id: checkout
         uses: actions/checkout@v2
 
       - name: "Git SSH"
+        id: git_ssh
         if: ${{ github.actor != 'dependabot[bot]' }}
         uses: linc-technologies/.github/.github/actions/git_ssh@master
         with:
           key: ${{ secrets.VCS_SSH_KEY }}
 
       - name: "Setup Node 10.21"
+        id: setup_node
         uses: actions/setup-node@v3
         with:
           node-version: 10.21
           cache: 'npm'
           cache-dependency-path: linced-*/package-lock.json
 
       - name: "Login to ACR"
+        id: login_to_acr
         uses: docker/login-action@v2
         with:
           registry: linced.azurecr.io
           username: ${{ secrets.AZURE_ACR_LINCED_USERNAME }}
           password: ${{ secrets.AZURE_ACR_LINCED_PASSWORD }}
 
       - name: "Install Dependencies (npm ci)"
-        if: contains(fromJson('["testing", "development", "staging", "production"]'), github.ref_name)
```

Comment:
@matthewhartstonge We're just disregarding this as it's always true?

Reply:
Nah, for any branch in order to lint we need the dependencies pulled down. We detected this as I had a PR being merged into another branch and the deps didn't install.

Other thing this PR does is builds will now only happen on PR - compared to happening on every time code was pushed to Github.